.github/workflows/ci.yaml

```diff
@@ -0,0 +1,134 @@
+name: build
+
+on:
+  push:
+    branches: [ testing ]
+  pull_request:
+    branches: [ testing ]
```

Comment:
Didn't we say, we deploy master ( = production) to kube first? The DB dump was from production and starting with the wrong db dump will confuse telegram integration

Reply:
I thought we deploy testing (master lacks a lot of recent developments) and starting from a prod db should not confuse the app as it would be the same as merging testing to master?

Reply:
works as well. @uubk just make sure you entered the bot token and name from `versusvirus.njsm.de` and not `versusvirus-testing.njsm.de`

Reply:
I had the same understanding as @ntruessel. so I used the bot token from `versusvirus.njsm.de`.